.circleci/prepare-kernels/prepare-legacy-sources.sh

```diff
@@ -4,6 +4,11 @@ set -eo pipefail
 TAG=$1
 BRANCH=$2
 
+if [[ -z "$TAG" && "$BRANCH" != "master" && ! -f "${WORKSPACE_ROOT}/pr-metadata/labels/build-legacy-probes" ]]; then
```

Comment:
We should add this change on the master branch as well if it is applicable.

Reply:
I will look into it.